docs/HowToGuides/GettingStarted.md

```diff
@@ -305,16 +308,25 @@ It is even possible to integrate the Ninja build into Xcode. For details on how
   In many situations, there are several errors, so scrolling further back
   and looking at the first error may be more helpful than simply looking
   at the last error.
-- Check if others have encountered the same issue on the Swift forums or on
-  [Swift repository 'Issues' tab][Swift Issues].
-- Create a new Swift forums thread in the Development category. Include
-  information about your configuration and the errors you are seeing.
+- Check if others have encountered the same issue on the [Swift Forums](https://forums.swift.org/c/development/compiler) or on
+   [Swift repository 'Issues' tab][Swift Issues].
+  - Here is a list of threads that describe common issues:
```

Comment:
Maybe merge this into the parent bullet?

Reply:
+1